Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -9,6 +9,15 @@ public class PlayerControls : MonoBehaviour
 	[SerializeField] float _interactCheckRadius = 0.2f;
 
 	PlayerActor _actor;
+
+	[Tooltip( "Probably straight up" )]
+	[SerializeField] Vector3 _respawnOffset = new Vector3();
+	[ReadOnly( "Respawn Position" )]
+	[SerializeField] Vector3 _respawnPosition = new Vector3();
+	Quaternion _respawnRotation = new Quaternion();
```

Comment:
`Quaternion` and `Vector3` are structs, so they're functionally identical. That being said, it would still be a good idea to to use `Quaternion.Identity` and `Vector3.Zero` because they're more explicit about what values are being set.